src/csharp/Microsoft.Spark/Sql/RowConstructor.cs

```diff
@@ -65,15 +65,6 @@ public object construct(object[] args)
                 s_schemaCache = new Dictionary<string, StructType>();
             }
 
-            // When a row is ready to be materialized, then construct() is called
```

Comment:
This is actually a breaking change, but @suhsteve is updating the worker version in his PR: https://github.com/dotnet/spark/pull/387/files